aws/v11.5.2/release.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
+  creationTimestamp: null
+  name: v11.5.2
+spec:
+  apps:
+  - name: cert-exporter
+    version: 1.2.3
+  - componentVersion: 0.9.0
+    name: cert-manager
+    version: 1.0.8
+  - name: chart-operator
+    version: 0.12.4
+  - componentVersion: 1.16.5
+    name: cluster-autoscaler
+    version: 1.16.0
+  - componentVersion: 1.6.5
+    name: coredns
+    version: 1.2.0
+  - componentVersion: 0.7.2
+    name: external-dns
+    version: 1.2.2
+  - componentVersion: 3.5.0
+    name: kiam
+    version: 1.3.0
+  - componentVersion: 1.9.5
+    name: kube-state-metrics
+    version: 1.1.0
+  - componentVersion: 0.3.3
+    name: metrics-server
+    version: 1.1.0
+  - name: net-exporter
+    version: 1.9.0
+  - componentVersion: 0.18.1
+    name: node-exporter
+    version: 1.2.0
+  components:
+  - name: app-operator
+    version: 1.0.0
+  - name: aws-cni
+    version: 1.6.0
+  - name: aws-operator
+    releaseOperatorDeploy: true
+    version: 8.7.6
+  - name: calico
+    version: 3.15.1
+  - name: cert-operator
+    releaseOperatorDeploy: true
+    version: 0.1.0
+  - name: cluster-operator
+    releaseOperatorDeploy: true
+    version: 2.3.2
+  - name: containerlinux
+    version: 2512.2.1
+  - name: etcd
+    version: 3.4.9
+  - name: kubernetes
+    version: 1.16.13
+  date: "2020-08-14T10:16:08Z"
```

Comment:
Does the date have to stick with the day we want to release it?

Reply:
Yep

Reply:
Thanks 🙂, I was not sure about that